Assets/AssetReview/UltimateGridsEngine/Common/Runtime/Pathfinding/GridPathfinder.cs

```diff
@@ -49,11 +50,16 @@ public class GridPathfinder : MonoBehaviour {
     /// On Update, we draw the path if needed, determine the next waypoint, and move to it if needed
     /// </summary>
     protected virtual void Update() {
+        timer -= Time.deltaTime;
         if (m_DestinationTile == null) {
             return;
         }
-        DetermineNextPathpoint();
-        MoveGridObject();
+        if (timer <= 0)
+        {
+            DetermineNextPathpoint();
+            MoveGridObject();
+            timer = 2f;
```

Comment:
I know this is temp code that will change in the future. But we will eventually need to make constants like this `timer = 2f;` a constant or at least something that is serialized that we can edit in the inspector.

Reply:
I am pretty sure we won't need to do that because the movement code takes in an animation and makes the timing go off that but if it doesn't then we can make it a constant.